CONTRIBUTING.md

```diff
@@ -124,9 +124,38 @@ The maintenance branch's HEAD will be the target for our release tag:
 $ git tag -a -m "Release X.Y.Z" X.Y.Z
 ```
 
-### Push tags
+### Update branch-alias for dev-master (major and minor releases only)
+
+Initial versions of new major or minor releases will likely be made on the
+master branch directly instead of a maintenance branch. In this case, the
+`branch-alias` for `dev-master` in `composer.json` should be updated. For
+example, after tagging a new 1.10.0 release on the master branch,
+`composer.json` in the master branch may still read:
```

Comment:
Regardless of when releases are tagged, the `branch-alias` needs to be changed once we branch off. For example, after branching `v1.10`, the `branch-alias` for `dev-master` should be changed to `1.11.x`.

Reply:
Good point. I'll move this to a separate section that talks about branching in that case.